odoo/addons/test_new_api/tests/test_onchange.py

```diff
@@ -962,6 +962,13 @@ def test_copy(self):
         self.assertEqual(record.line_ids.mapped('foo'), ['foo1', 'bar'])  # copied
         self.assertEqual(record.tag_ids, tag_foo + tag_bar)  # copied
 
+    def test_copy_batch(self):
+        partners = self.env['test_new_api.partner'].create([{'name': f'Partner {i}' for i in range(3)}])
+        with self.assertQueryCount(3):
```

Comment:
What are the three queries for?  Maybe you should have 10 records to show that it's not linear.